lib/src/generator_options.dart

```diff
@@ -37,6 +43,8 @@ class GeneratorOptions {
   final String testMethodName;
   final String testerType;
   final String testerName;
+  final bool enableHooks;
```

Comment:
Please either rename the property to `addHooks` or change the name in the configuration file.